src/components/ui/buttons/Button.stories.tsx

```diff
@@ -1,84 +1,79 @@
-import {
-  ComponentMeta,
-  ComponentStory,
-  ComponentStoryObj,
-} from '@storybook/react'
+import {Meta, StoryObj} from '@storybook/react'
 import {Button} from './Button'
 import {Row} from '@/components/ui/layout/Row'
+import pressableArgTypes from '@/storybook/utils/pressable-arg-types'
 
-export default {
+const meta: Meta<typeof Button> = {
+  argTypes: pressableArgTypes,
   component: Button,
-  argTypes: {
-    onPress: {
-      action: 'onPress',
-    },
-  },
-} as ComponentMeta<typeof Button>
+}
+
+export default meta
 
-export const Primary: ComponentStoryObj<typeof Button> = {
+type Story<T extends typeof Row | void = void> = StoryObj<typeof Button & T>
+
+export const Primary: Story = {
   args: {
-    label: 'Label',
+    label: 'Primary',
     variant: 'primary',
   },
 }
 
-export const Secondary: ComponentStoryObj<typeof Button> = {
+export const Secondary: Story = {
   args: {
-    label: 'Label',
+    label: 'Secondary',
     variant: 'secondary',
   },
 }
 
-export const Tertiary: ComponentStoryObj<typeof Button> = {
-  args: {
-    label: 'Label',
-    variant: 'tertiary',
-  },
-}
-
-export const _Inline: ComponentStory<typeof Button & typeof Row> = ({
-  align,
-}) => (
-  <Row align={align}>
-    <Button label="Inline button" />
-  </Row>
-)
-
-_Inline.args = {
-  align: 'start',
-}
-
-export const __Alignment: ComponentStory<typeof Button & typeof Row> = ({
-  align,
-  gutter,
-}) => (
-  <Row
-    align={align}
-    gutter={gutter}>
-    <Button label="First button" />
-    <Button
-      label="Second button"
-      variant="secondary"
-    />
+export const Tertiary: Story = {
```

Comment:
is er een reden dat deze tertiary een andere opzet heeft dan de primary en secondary? met een eigen render in plaats van alleen args?

Reply:
Slorrrdig